EIPS/eip-1.md

```diff
@@ -95,6 +93,8 @@ A PR moving an EIP from Last Call to Final SHOULD contain no changes other than
 
 **Living** - A special status for EIPs that are designed to be continually updated and not reach a state of finality. This includes most notably EIP-1.
 
+A PR moving an EIP from one status to another MUST contain no changes other than the status update. Any other changes MUST be submitted in one or more separate PRs. When the content of an EIP changes, the git branch of any PR that changes the status of that EIP MUST be updated to reflect the current version.
```

Comment:
```suggestion
A PR moving an EIP from one status to another MUST contain no changes other than the status update. Any other changes MUST be submitted in one or more separate PRs. When the content of an EIP changes, the git branch of any PR that changes the status of that EIP MUST be rebased to reflect the current version.
```
updating changes via merge messes up the commit list on the PR, would prefer pure rebasing and hence the tighter language